### ISSUE TITLE
Netflix addon only allows selection of 2-channel audio

### ISSUE THREAD

**Issue by greenmr** [CastagnaIT/plugin.video.netflix#482]
I am running the Netflix 0.16.4 addon in Kodi 18.5 on a Fire TV Stick 4K. I have Kodi configured for audio passthrough to my older Sony 5.1 audio DD/DTS SPDIF receiver, which works perfectly with all my other Kodi addons. My Netflix subscription is Premium Ultra HD and both the native Netflix app on the Firestick and the Netflix app on my Sony BDP-S6500 Bluray player play 5.1 audio correctly through the surround receiver.  The problem is that no matter how I try to configure it, the Kodi Netflix addon simply will not even show any 5.1 audio streams. I can see all the same language choices as with the two dedicated players, but in the Kodi Netflix addon they all show as "AAC - 2 channels" and all the 5.1 audio versions are missing.

Although I doubt it will reveal much, [here is my log file](https://pastebin.com/uprm2HPN) which includes starting playback of a Netflix show I know for certain has 5.1 audio.

**Comment by CastagnaIT** [comment]
check in addon expert setting and turn on "Enable dolby digital plus"

**Comment by greenmr** [comment]
> check in addon expert setting and turn on "Enable dolby digital plus"

Thanks for your reply. That worked, but it doesn't make sense. I started with it set that way, but it wasn't providing any 5.1 audio streams. In the process of trying to solve it I turned that setting off and had left it that way. Turning it back on as per your suggestion got it working. No idea why it wasn't working before when I had it turned on, but it is now and that's all that matters. Thanks again
